pyproject.toml

```diff
@@ -165,7 +165,7 @@ doc = [
     "pyzmq!=24.0.0",
     "ipython!=8.7.0",
     "selenium",
-    "intersphinx_registry",
+    "intersphinx_registry>=0.2405.27",
```

Comment:
@Carreau I added this pin, so let's keep this version schema if possible :) Thanks for the replies!